values.yaml

```diff
@@ -211,6 +211,7 @@ ingress:
 
 httpProxy: ""
 httpNoProxy: ""
+logRetention: "30"  # Days to keep application logs
```

Comment:
Did you want to put quotes around the value here? 

I noticed that other `int` values in Values.yaml are not quoted. I also noticed that in synopsysctl we are setting it as an integer. 

I do not think this is an issue since you use quote, `{{ .Values.logRetention | quote}}`, in configmap-user.yaml... but I thought I would ask.

Reply:
Oh, no need for quotes. I'll remove.